nengo_gui/components/netgraph.py

```diff
@@ -179,23 +170,20 @@ def _reload(self, code=None):
 
                 self.uids[uid] = new_item
 
-        self.to_be_expanded.append(model)
+        self.to_be_expanded.append(self.sim.model)
+
+        t_uids = [self.sim.get_uid(c.template) for c in self.sim.components]
```

Comment:
Yeah, a better name would be good...  probably ```template_uids``` or even ```old_template_uids```, since it's the uids of the templates for the old components.  That's also why this code is up here -- I have to compute these before wiping out the names and loading everything new, so this is as close as I could get it to where they are used.

That said, the plan is to wipe out the whole template thing...  still, a better comment/name here would be good.  :)